conans/client/generators/pkg_config.py

```diff
@@ -86,7 +87,10 @@ def _pc_file_content(self, name, cpp_info, requires_gennames):
         description = cpp_info.description or "Conan package: %s" % name
         lines.append("Description: %s" % description)
         lines.append("Version: %s" % cpp_info.version)
-        libdirs_flags = ["-L${%s}" % name for name in libdir_vars]
+        if quote:
```

Comment:
without this, meson fails with obscure errors:
```
"cl" "-Idemo.exe.p" "-I." "-I.." "-IC:/temp/tmp5zs5278uconans/path" "-DNDEBUG" "/MD" "/nologo" "/showIncludes" "/W2" "/EHsc" "/O2" "/Ob2" "/Gw" "with" "spaces/data/hello/0.1/_/_/package/6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7/include" "/Fddemo.exe.p\main.cpp.pdb" /Fodemo.exe.p/main.cpp.obj "/c" ../main.cpp
cl : Command line warning D9024 : unrecognized source file type 'with', object file assumed
cl : Command line warning D9027 : source file 'with' ignored
cl : Command line warning D9024 : unrecognized source file type 'spaces/data/hello/0.1/_/_/package/6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7/include', object file assumed
cl : Command line warning D9027 : source file 'spaces/data/hello/0.1/_/_/package/6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7/include' ignored
../main.cpp(6): fatal error C1083: Cannot open include file: 'hello.h': No such file or directory
```
I am not sure why didn't we run into this before, but I assume quotes are needed to properly generate compiler command line

Reply:
pc file generated:
```
prefix=C:/temp/tmp5zs5278uconans/path with spaces/data/hello/0.1/_/_/package/6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7
libdir=${prefix}/lib
includedir=${prefix}/include

Name: hello
Description: <Description of Hello here>
Version: 0.1
Libs: -L${libdir} -lhello 
Cflags: -I${includedir}
```